deps.bzl

```diff
@@ -217,9 +217,9 @@ def daml_deps():
         # This should be kept in sync with the grpc version we get from Nix.
         http_archive(
             name = "com_github_grpc_grpc",
-            strip_prefix = "grpc-1.43.0",
-            urls = ["https://github.com/grpc/grpc/archive/v1.43.0.tar.gz"],
-            sha256 = "9647220c699cea4dafa92ec0917c25c7812be51a18143af047e20f3fb05adddc",
+            strip_prefix = "grpc-1.44.0-pre2",
```

Comment:
v1.44.0 is out now. Can we avoid using a pre-release?

Reply:
sure - will update -- it wasn't there as of yesterday yet :)

Reply:
Don't you need to update nixpkgs as well? The comment above says to keep them in sync.

Reply:
doesn't look like github.com/grpc/grpc official version of 1.44.0 is out yet 
<img width="1423" alt="image" src="https://user-images.githubusercontent.com/25179017/151845533-ba35af25-e0dc-4936-843b-4e14f0bcd401.png">

which nixpkgs needs to be kept in sync and how can we determine which version of grpc is part of that nixpkgs version?

Reply:
i reverted back to 1.43.0 as that looks to be what the nixpkgs-unstable we are using is at